nsxt/resource_nsxt_policy_lb_pool.go

```diff
@@ -491,17 +501,22 @@ func resourceNsxtPolicyLBPoolRead(d *schema.ResourceData, m interface{}) error {
 		return handleReadError(d, "LBPool", id, err)
 	}
 
+	if _, ok := d.GetOk("active_monitor_path"); ok {
+		// deprecated value was set
+		if obj.ActiveMonitorPaths != nil {
+			d.Set("active_monitor_path", obj.ActiveMonitorPaths[0])
+		}
+	} else {
+		// resource uses lists attribute (non-deprecated) or attribute
```

Comment:
if `active_monitor_path` was set before, and was deleted now, would it be nullified in the state?

Reply:
I hope I understand correctly your question.
I think the answer is yes, but I'm not sure if you want the attribute to be cleared or removed from the state.
The attribute is cleared, and I've also added a check in acceptance tests to verify that.

Resource change: 

```
  # nsxt_policy_lb_pool.test will be updated in-place
  ~ resource "nsxt_policy_lb_pool" "test" {
      - active_monitor_path      = "/infra/lb-monitor-profiles/default-http-lb-monitor" -> null
      + active_monitor_paths     = [
          + "/infra/lb-monitor-profiles/default-http-lb-monitor",
          + "/infra/lb-monitor-profiles/default-icmp-lb-monitor",
        ]
        id                       = "c756df01-a22f-48e9-95a3-4dd0d9e7cfd9"
        # (9 unchanged attributes hidden)

        # (4 unchanged blocks hidden)
    }
```

State after change:

```     
"instances": [
        {
          "schema_version": 0,
          "attributes": {
            "active_monitor_path": "",
            "active_monitor_paths": [
              "/infra/lb-monitor-profiles/default-http-lb-monitor",
              "/infra/lb-monitor-profiles/default-icmp-lb-monitor"
            ],
            "algorithm": "IP_HASH",
            "description": "yyy",
            "display_name": "xxx",
[...]
```